roles/grafana_cloud_operator/defaults/main.yml

```diff
@@ -23,3 +23,16 @@ grafana_cloud_operator_alertmanager_secret_name: alertmanager-main
 
 # SyncSet details
 grafana_cloud_operator_syncset_namespace: grafana-cloud-operator
+
+
+## SLO Dashboard
```

Comment:
Yes, ideally we shouldn't have them here. But This won't work for anyone else, until they have API key to use it

Reply:
# Preferable to make it parameterised so that one can provide input - maybe in the Config CR?

Yes I think that would make more sense. But it would polute the config CR

Reply:
No, this is config required from a user in order to setup SLO Dashboard. They would be part of a custom resource.